jetty-core/jetty-core-test-webapps/jetty-core-demo-webapp/src/main/assembly/core-webapp.xml

```diff
@@ -0,0 +1,39 @@
+<assembly xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.0 http://maven.apache.org/xsd/assembly-1.1.0.xsd">
+  <id>core-webapp</id>
```

Comment:
All of our other demos have the context xml file included in the zip file that is produced, but this one does not. There is the equivalent of a context xml file for it in code in the `DeployerTest` distribution test.

Reply:
This one has it in the zip file.
Not sure why you think it isn't in the zip file?

Reply:
Because the xml that is applied _first_ is the xml that is in code in the test `DeployerTest.testCoreDeployCustomHandlerInClasses` line 197. The `jetty-web.xml` is some secondary xml that gets applied I'm guessing.